pyproject.toml

```diff
@@ -248,6 +247,10 @@ max-complexity = 17
     "ANN401", # Dynamically typed expressions (typing.Any) are disallowed
 ]
 
+"infrahub_sdk/ctl/**/*.py" = [
+    "UP007",  # Use `X | Y` for type annotations
```

Comment:
If this was only related to `Typer` can you add this as a comment to indicate why we have this?

Reply:
Good point, I added a comment
Regarding Typer 0.15, I tried but the issue was still present